src/css/styles.scss

```diff
@@ -6,16 +6,17 @@ $title-font: "Sulphur Point", sans-serif;
 $white-in-cards: hsla(357, 15%, 95%, 1);
 $black-shadow-shadow: hsla(360, 5%, 0%, 1);
 $main-blue: hsla(209, 24%, 47%, 1);
-$seafoam-h2-hover: hsla(148, 38%, 65%, 1);
 $green: hsla(151, 48%, 40%, 1);
```

Comment:
was this always here I have this as something else on my branch

Reply:
This is what I got to work with when I first pulled that down, I love the color! I wish I could find a better way to incorporate it that still got a passing contrast score!